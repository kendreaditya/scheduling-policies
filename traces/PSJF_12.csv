1,0,100
2,0,101
3,201,200
4,201,1000
5,201,1
6,400,2
7,400,3
8,400,100
9,400,1
10,600,2
11,600,3
12,600,4
13,1000,0
14,1000,0
15,1000,0
16,1000,0
17,1000,0
18,1000,0
19,1000,0
20,1000,0
21,2000,100
22,3000,50
23,4000,25
24,5000,12
25,6000,6
26,7000,3
27,8000,2
28,9000,1
29,10000,0
30,11000,0
