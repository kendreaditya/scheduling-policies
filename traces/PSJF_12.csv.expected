1,100
2,201
3,508
4,1517
5,202
6,403
7,406
8,506
9,401
10,602
11,605
12,609
13,1000
14,1000
15,1000
16,1000
17,1000
18,1000
19,1000
20,1000
21,2100
22,3050
23,4025
24,5012
25,6006
26,7003
27,8002
28,9001
29,10000
30,11000
